### ISSUE TITLE
Ability to rollback a migration?

### ISSUE THREAD

**Issue by andrejewski** [austinkelleher/dynograte#3]
If there is a problem with a new migration, is there a mechanism to define rollback complements to our migrations?

**Comment by philidem** [comment]
We talked about your suggestion and I think the migrations scripts should look something like:

```javascript
// upgrade
exports.up = (dynamodb) => {
};

// rollback
exports.down = (dynamodb) => {
};
```

**Comment by andrejewski** [comment]
Yeah, that's how it is done with [other](https://github.com/tj/node-migrate#creating-migrations) [tools](https://db-migrate.readthedocs.io/en/latest/Getting%20Started/usage/).

**Comment by austinkelleher** [comment]
@andrejewski I'm still working on this and should have some improvements up soon.
